pandas/core/generic.py

```diff
@@ -4869,6 +4869,10 @@ def bfill(self, axis=None, inplace=False, limit=None, downcast=None):
     _shared_docs['replace'] = ("""
         Replace values given in 'to_replace' with 'value'.
 
+        Values of the DataFrame or a Series are being replaced with
```

Comment:
Not sure this extended description is adding much. Better served to make mention of how this can replace values with a dynamic set of inputs like dicts

Reply:
done, thank you for the suggestion